notebooks/Satellite_Data/GOES_Interactive_Plot.ipynb

```diff
@@ -19,7 +19,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Shouldn't we be clearing all output before committing?

Reply:
I've been careful on the exercise notebooks. This is a demo notebook so I could go either way. Web rendering = good